src/platform/ios.rs

```diff
@@ -270,3 +308,17 @@ bitflags! {
             | ScreenEdge::BOTTOM.bits() | ScreenEdge::RIGHT.bits();
     }
 }
+
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
+#[allow(dead_code)]
+#[repr(isize)]
+pub enum UIStatusBarStyle {
```

Comment:
The convention in this file is to call this `StatusBarStyle`, without the `UI` prefix.

Also, I'd prefer to not have a `repr` on this, instead we should have an internal `UIStatusBarStyle` that this maps to.